tools/debian/rules

```diff
@@ -5,7 +5,7 @@ DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 # we need an apparmor profile for >= 4.0; this exists in Ubuntu >= 24.04, but not in Debian yet
 PRE4AA = $(filter $(shell . /etc/os-release; echo $${VERSION_ID:-unstable}),22.04 11 12 unstable)
 
-CONFIG_OPTIONS := --disable-pcp
+CONFIG_OPTIONS :=
```

Comment:
This can be fully removed, I introduced this line and it is unneeded now. See 112ce31a06deee0d63cc255e6ce26cacfaeb2d9a